scene/3d/soft_body_3d.cpp

```diff
@@ -683,6 +682,11 @@ void SoftBody3D::pin_point(int p_point_index, bool pin, const NodePath &p_spatia
 	}
 }
 
+void SoftBody3D::_pin_point_deferred(int p_point_index, bool pin, const NodePath p_spatial_attachment_path) {
+	pin_point(p_point_index, true, p_spatial_attachment_path);
```

Comment:
Either the argument `pin` in the function signature is not needed or the value `true` in the call to `pin_point` in the function can be replaced with the argument `pin`. As it is, the `pin` argument is unused.

Reply:
Oversight on my part, thanks.

Reply:
This appears to have been fixed in favor of passing the `pin` argument into `pin_point` from the deferred call function.
I'll have to test if this PR fixes the issue on the next beta/RC release after it is merged.